frontend/src/components/User/User.js

```diff
@@ -15,58 +17,38 @@ export const User = props => {
         event.preventDefault();
         axios.post(global.config.BACKEND_URL + "/account/following/newFollower/" + props.username)
             .then(
-                (response) => {
-                    setFollowing(true);
-                },
+                (response) => { setFollowing(true); },
                 (error) => {
-                    if (error.response)
-                    {
-                        setErrorMsg(error.response.data.message);
-                    } 
-                    else
-                    {
-                        setErrorMsg("An unknown error occurred");
-                    }
+                    if (error.response) { setErrorMsg(error.response.data.message); }
+                    else { setErrorMsg("An unknown error occurred"); }
                 }
             )
     };
 
     const unfollow = (event) => {
         event.preventDefault();
-        axios.delete(global.config.BACKEND_URL + "/account/following/followerRemoval/"+ props.username)
+        axios.delete(global.config.BACKEND_URL + "/account/following/followerRemoval/" + props.username)
             .then(
-                (response) => {
-                    setFollowing(false);
-                },
+                (response) => { setFollowing(false); },
```

Comment:
No need for response parameter

Reply:
This part is going to be removed once I merged the updated master back into my branch. It was fixed and removed during @daniela-venuta 's PR